flow/probes/ebpf/Makefile

```diff
@@ -0,0 +1,14 @@
+LLC ?= llc
+CLANG ?= clang
+
+all: flow_table.o
+
+%.o: %.c
+	$(CLANG) \
+		-I$(GOPATH)/src/github.com/safchain/goebpf \
```

Comment:
Probably as some part of the iovisor implementation comes from my implementation so not so different
https://github.com/iovisor/gobpf/blob/master/elf/elf.go#L4

I just resubmitted what I did one year ago